cloudformation/proxy-service.yaml

```diff
@@ -296,11 +315,6 @@ Resources:
                 Resource:
                   - !Sub "arn:aws:s3:::${EcsAppGeoipS3Bucket}"
                   - !Sub "arn:aws:s3:::${EcsAppGeoipS3Bucket}/*"
-              - Effect: Allow
```

Comment:
@mmiermans lets leave this in. If we remove it then we need to time deploys perfectly, but if we leave it we can deploy the code change at will.